src/Admin/EventAdmin.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Admin;
+
+use App\Entity\Event;
+use Sulu\Bundle\AdminBundle\Admin\Admin;
+use Sulu\Bundle\AdminBundle\Admin\Navigation\NavigationItem;
+use Sulu\Bundle\AdminBundle\Admin\Navigation\NavigationItemCollection;
+use Sulu\Bundle\AdminBundle\Admin\View\ToolbarAction;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewBuilderFactoryInterface;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewCollection;
+use Sulu\Component\Security\Authorization\PermissionTypes;
+use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
+
+class EventAdmin extends Admin
+{
+    const LIST_VIEW = 'app.event.list';
+    const ADD_FORM_VIEW = 'app.event.add_form';
+    const ADD_FORM_DETAILS_VIEW = 'app.event.add_form.details';
+    const EDIT_FORM_VIEW = 'app.event.edit_form';
+    const EDIT_FORM_DETAILS_VIEW = 'app.event.edit_form.details';
+
+    private ViewBuilderFactoryInterface $viewBuilderFactory;
+    private SecurityCheckerInterface $securityChecker;
+
+    public function __construct(
+        ViewBuilderFactoryInterface $viewBuilderFactory,
+        SecurityCheckerInterface $securityChecker
+    ) {
+        $this->viewBuilderFactory = $viewBuilderFactory;
+        $this->securityChecker = $securityChecker;
+    }
+
+    public function configureNavigationItems(NavigationItemCollection $navigationItemCollection): void
+    {
+        if ($this->securityChecker->hasPermission(Event::SECURITY_CONTEXT, PermissionTypes::EDIT)) {
+            $rootNavigationItem = new NavigationItem('app.events');
+            $rootNavigationItem->setIcon('su-calendar');
```

Comment:
Available icons are documented in the [Sulu Javascript Docs](https://jsdocs.sulu.io/latest/#icon).